src/registrar/models/utility/contact_error.py

```diff
@@ -1,2 +1,51 @@
+from enum import IntEnum
+
+
+class ContactErrorCodes(IntEnum):
+    """Used in the ContactError class for
+    error mapping.
+
+    Overview of contact error codes:
+        - 2000 CONTACT_TYPE_NONE
```

Comment:
random, just curious, why 2000?

Reply:
I decided to somewhat follow the pattern from RFC 5730 - though very loosely (not literally). Not opposed at all to changing it, I just needed something to use here
![image](https://github.com/cisagov/getgov/assets/141044360/4b6f0e66-2dd5-4597-aa78-56901cc0c0f6)

Reply:
The number doesn't matter so it's whatever you prefer. That is based on EPP standards so really happy if it matches or happy if it doesn't.